TravelFunds/2018.md

```diff
@@ -46,3 +46,4 @@ Waleed	|	Ashraf	|	JS Interactive and Collaborator Summit	|	Attendance, Collabora
 Saleh	|	Abdel Motaal	|	JS Interactive and Collaborator Summit	|	Attendance, Collaborate and Code & Learn	|	Vancouver BC, Canada	|	10Oct -14Oct 2018	|	US$1700	| 19Sep 2018 | https://github.com/nodejs/admin/pull/232 | | | | |	
 Anatoli	|	Papirovski	|	JS Interactive and Collaborator Summit	|	Attendance, Collaborate and Code & Learn	|	Vancouver BC, Canada	|	9Oct - 13Oct 2018	|	US$1633.53	|	26Sep 2018	|	https://github.com/nodejs/admin/pull/235 |		|		|		|		|	
 Dan | Shaw | JS Interactive and Collaborator Summit | Attendance, Collaborate and Code & Learn | Vancouver, BC, CAN | 9oct - 14oct 2018 | US$1820 | | https://github.com/nodejs/admin/pull/224 | | | | |
+Ujjwal | Sharma | Code & Learn, Bengaluru | Collaborate and Code & Learn | Bengaluru, India | 17th Nov 2018 | US$300 | 5th Sep 2018 | | | | | |
```

Comment:
Nit: The pull request date should be updated to 5th Nov 2018 I guess.

Reply:
@thefourtheye I thought this was optional. I could update this before landing?